QUERIES TASK.sql

```diff
@@ -0,0 +1,120 @@
+--1- every customer (E-Shop) 
+--wants to know who the top 3 customers buy from them
+SELECT c.Name as TOP_3_CUSTOMERS
+FROM Orders AS o
+JOIN Customers AS c
+ON o.CustomerID = c.CustomerID
+GROUP BY c.CustomerID
+ORDER BY COUNT(o.OrderID) DESC
+LIMIT 3;
+
+--2- show me the customers 
+-- who bought more than 100$ 
+-- on the last day 
+SELECT c.Name, o.Total_Amount, o.Order_Date
+FROM Customers AS c
+JOIN Orders AS o
+ON c.CustomerID = o.CustomerID
+WHERE o.Total_Amount > 100 AND o.Order_Date = CURRENT_DATE - INTERVAL '1 day';
```

Comment:
if the customer had 3 orders (50, 60, 70)$ in one day, in this case the customer will not be in the list